explainshell/web/static/js/es.js

```diff
@@ -568,15 +568,23 @@ function drawgrouplines(commandselector, options) {
         var link = links[i];
 
         console.log('handling', $(link.option).text());
-        var rr = link.option.getBoundingClientRect(),
-            spanmid = rr.left + rr.width / 2,
-            rrright = rr.right - strokewidth;
+        var commandRect = link.option.getBoundingClientRect(),
+            spanmid = commandRect.left + commandRect.width / 2,
+            commandRight = commandRect.right - strokewidth;
 
-        var path = new espath();
+        link.starty = commandRect.bottom - commandWrapperRect.top + 1;
+        commandOffsettToCanvas = marginBetweenCommandAndCanvas - link.starty
+
+        // points for marker under command
+        link.paths.push(new espath()
+          .addpoint(commandRect.left, 0)
+          .addpoint(commandRect.left, 5)
+          .addpoint(commandRight, 5)
+          .addpoint(commandRight, 0)
+        );
 
-        link.paths.push(new espath().addpoint(rr.left, 0).addpoint(rrright, 5));
-        link.paths.push(new espath().addpoint(rrright, 5).addpoint(rrright, 0));
-        path.addpoint(rr.left + rr.width / 2, 5); // 3
+        var path = new espath();
```

Comment:
Yes we continue to add points here
https://github.com/idank/explainshell/pull/166/files?diff=unified#diff-ab5c511b11e469baa7210dae1051b2bdR602
https://github.com/idank/explainshell/pull/166/files?diff=unified#diff-ab5c511b11e469baa7210dae1051b2bdR605
https://github.com/idank/explainshell/pull/166/files?diff=unified#diff-ab5c511b11e469baa7210dae1051b2bdR617
https://github.com/idank/explainshell/pull/166/files?diff=unified#diff-ab5c511b11e469baa7210dae1051b2bdR627
https://github.com/idank/explainshell/pull/166/files?diff=unified#diff-ab5c511b11e469baa7210dae1051b2bdR630
https://github.com/idank/explainshell/pull/166/files?diff=unified#diff-ab5c511b11e469baa7210dae1051b2bdR642

and then we add path to link
https://github.com/idank/explainshell/pull/166/files?diff=unified#diff-ab5c511b11e469baa7210dae1051b2bdR646